src/Evolution/Systems/RelativisticEuler/Valencia/Characteristics.hpp

```diff
@@ -74,8 +75,131 @@ std::array<DataVector, Dim + 2> characteristic_speeds(
     const tnsr::i<DataVector, Dim>& normal) noexcept;
 // @}
 
-namespace Tags {
+/*!
+ * \brief Right eigenvectors of the Valencia formulation
```

Comment:
Possibly in the brief, but ideally also in the main body, say what this means: these vectors let you transform between the conserved and characteristic variables of the system.